src/apiV2/client.ts

```diff
@@ -10,38 +10,50 @@ import { DEFAULT_DOMAIN } from "../common/constants";
 import { generateStream } from "../common/generateStream";
 import { EventBuffer } from "./EventBuffer";
 
-const convertReranker = (reranker?: StreamQueryConfig["search"]["reranker"]) => {
-  if (!reranker) return;
-
-  if (reranker.type === "none") {
+const convertReranker = (reranker?: StreamQueryConfig["search"]["reranker"]): Query.NoneReranker | Query.ChainReranker => {
+  if (reranker?.isEnabled) {
     return {
-      type: reranker.type
-    };
-  }
+      type: "chain",
+      rerankers: reranker?.names?.split(",").map((name: string) => {
```

Comment:
Where can an external user get a list of these names? When I hit the [List Rerankers API](https://docs.vectara.com/docs/rest-api/list-rerankers) the names look different from the expected ones here.

Reply:
we can mention in the docs probably ? I got it from docs that we are using in the vectara-answer.

Reply:
I would recommend using the names in the list API or relying on ids as opposed to having to add yet a new name mapping to the docs. We should ideally converge on a single source of truth.